src/hotspot/share/cds/metaspaceShared.cpp

```diff
@@ -141,11 +141,17 @@ size_t MetaspaceShared::core_region_alignment() {
 }
 
 static bool shared_base_valid(char* shared_base) {
-#ifdef _LP64
```

Comment:
Looks like this PR contains #16727 ?

Reply:
Its done as dependent PR. If all things worked, 16272 changes should not have showed up here. Not sure what the problem is.

Reply:
It sometimes requires merging in the latest master branch.